cleverhans/attacks.py

```diff
@@ -25,16 +363,17 @@ def fgsm(x, predictions, eps, back='tf', clip_min=None, clip_max=None):
                     value for components of the example returned
     :return: a tensor for the adversarial example
     """
+    warnings.warn("attacks.fgsm is deprecated and will be removed on 09-27-17."
+                  " Instantiate an attack object from FastGradientMethod.")
     if back == 'tf':
```

Comment:
might make more sense to wrap the FastGradientMethod class and then that handles the choice of backend for you